client/src/components/Header.js

```diff
@@ -5,13 +5,14 @@ class Header extends React.Component {
   static propTypes = {
     isLoggedIn: PropTypes.bool.isRequired,
     logoutUser: PropTypes.func.isRequired,
-    user: PropTypes.object
+    user: PropTypes.object,
+    isMusician: PropTypes.bool.isRequired
   };
 
   render() {
     const loggedIn = (
       <div>
-        Hello, {this.props.user.email}
+        Hello, <Link to={this.props.isMusician ? '/musicianProfile' : 'organizerProfile'}>{this.props.user.email}</Link>
```

Comment:
Let's change `'organizerProfile'` to be an absolutely referenced link as well i.e. `'/organizerProfile'`. That way if we the user is on a page that has a nested route e.g. `/events/1/attendees` the link will go to `/organizerProfile` instead of `/events/1/organizerProfile`